sound/audioclass.py

```diff
@@ -95,34 +55,27 @@ def GetAudioClass(self, data):
         else:
             d.popleft();d.append(self.Decibel)
             self.loop += 1
+        change = data.suddenchange
+        print "sudden sound change value {}".format(data.suddenchange)
+    	
+        loud = '(StateLink Sudden sound change value (NumberNode "' + \
```

Comment:
It needed to be `(StateLink (AnchorNode "Sudden sound change value") (NumberNode ...`